### ISSUE TITLE
Flutter v1.2.0

### ISSUE THREAD

**Issue by Mariluk** [dart-lang/reflectable#163]
After update Flutter to version 1.2.0 and run `flutter packages upgrade` i get error: 

The current Dart SDK version is 2.1.1-dev.3.2.flutter-f4afaee422.

Because every version of flutter_test from sdk depends on source_span 1.5.3 and reflectable >=2.0.2 depends on source_span >=1.0.0 <1.5.0, flutter_test from sdk is incompatible with reflectable >=2.0.2.
And because reflectable <2.0.2 requires SDK version >=1.8.0 <2.0.0-dev.infinity, flutter_test from sdk is incompatible with reflectable.
So, because netmetr_flutter depends on both reflectable any and flutter_test any from sdk, version solving failed.
pub upgrade failed (1)

`flutter doctor`

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel unknown, v1.2.0, on Linux, locale cs_CZ.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] Android Studio (version 3.3)
[✓] Connected device (1 available)

• No issues found!

**Comment by Mariluk** [comment]
Flutter v1.1.9 works ok.

**Comment by eernstg** [comment]
It looks like a very small adjustment will help, for now. You could try to use this entry in pubspec.yaml for reflectable:

```yaml
  reflectable:
    git:
      url: git://github.com/dart-lang/reflectable.git
      ref: version_fix_feb19
```

I'll get that landed and published such that you can return to use a published version.

**Comment by Mariluk** [comment]
It works. Thank you.

**Comment by eernstg** [comment]
There will be further issues when I adjust the version constraints of reflectable to allow for analyzer versions up to '<0.36.0', but that seems to involve deprecation of some methods that reflectable depends on in substantial ways, so that will take more work. I've requested some more information about this, so I hope it'll be resolved soon. In any case, Flutter does not yet require those analyzer versions.

**Comment by eernstg** [comment]
I just published the updates as version [2.0.10](https://pub.dartlang.org/packages/reflectable). I'll delete the branch soon, so please revert to use something like `reflectable: ^2.0.10` in 'pubspec.yaml.

**Comment by eernstg** [comment]
Thanks! Deleting the branch and closing the issue now.
